lua/damagelogs/shared/events/c4.lua

```diff
@@ -90,6 +99,8 @@ function event:ToString(v)
 		return string.format(TTTLogTranslate(GetDMGLogLang, "C4Planted"), v[2], Damagelog:StrRole(v[3]))
 	elseif v[1] == 5 then
 		return string.format(TTTLogTranslate(GetDMGLogLang, "C4Destroyed"), v[2], Damagelog:StrRole(v[3]), v[5])
+	elseif v[1] == 6 then
+		return string.format(TTTLogTranslate(GetDMGLogLang, "C4Exploded"), v[2])
```

Comment:
An innocent or detective can also use a C4. It would be good to add `Damagelog:StrRole(v[3])` for the role as `%s` parameter to `C4Exploded`:
`C4Exploded = "C4 of %s [%s] exploded."`

Reply:
I tried to follow the style of other C4 lines, where role of the planter wasn't mentioned. I'll edit my PR.